lib/ReplaceSource.js

```diff
@@ -55,25 +54,23 @@ class ReplaceSource extends Source {
 			throw new Error("str must be a string, but is a " + typeof str + ": " + str);
 		this._sortReplacements();
 		var result = [str];
-		this.replacements.forEach(function(repl) {
+		this.replacements.forEach(repl => {
 			var remSource = result.pop();
 			var splitted1 = this._splitString(remSource, Math.floor(repl[1] + 1));
 			var splitted2 = this._splitString(splitted1[0], Math.floor(repl[0]));
 			result.push(splitted1[1], repl[2], splitted2[0]);
 		}, this);
```

Comment:
i think you dont need to pass this here into `forEach`

Reply:
Yes that's correct. Thanks.